### ISSUE TITLE
Too many new(bit.Int),Huge pressure when process GC. how can i avoid that?

### ISSUE THREAD

**Issue by lvhuat** [shopspring/decimal#208]
As the title discribed

**Comment by mwoss** [comment]
Hi! Do you mean the `decimal` library is making too many calls to `new(big.Int)` overall or there are too many calls to `new(big.Int)` in your code due to usage of this library? Can you elaborate?

**Comment by lvhuat** [comment]
@mwoss  yes! Not only `new(big.Int)` is call too many times, but also slice field `abs` in `decimal.Decimal`.

**Comment by mwoss** [comment]
Thanks for letting us know. I will look at this in the future, as there are many more important issues than that currently. 
Also, this library is mainly aiming for correctness and precision of calculations rather than speed, so performance issues are resolved at the end.
